lib/stripe/customer.rb

```diff
@@ -54,6 +54,10 @@ def delete_discount
       refresh_from({ :discount => nil }, opts, true)
     end
 
+    def deleted?
+      self.respond_to?(:deleted)
```

Comment:
If I'm reading this correctly, it looks like you might just be checking whether the object responds to a call to `deleted`. This is good, but you'd probably also want to check the value inside `#deleted` as well.